lib/Page.js

```diff
@@ -62,10 +63,43 @@ function calculateNewBaseUrl(filePath, root, lookUp) {
   return calculate(filePath, []);
 }
 
+function insertFlexDiv(pageData) {
+  const $ = cheerio.load(pageData);
+  const footers = $('footer');
+
+  footers.each((index, footer) => {
+    if (!($(footer).attr('id') === 'flexDiv')) {
+      $(footer).before(FLEX_DIV_HTML);
+    }
+  });
+  return new Promise((resolve) => {
+    resolve($.html());
+  });
+}
+
 function unique(array) {
   return array.filter((item, pos, self) => self.indexOf(item) === pos);
 }
 
+function removeDuplicateFooter(pageData) {
+  const $ = cheerio.load(pageData);
+  const footers = $('footer');
+  const flexDivs = $('#flexDiv');
+  if (footers.length > 1) {
+    const lastFooter = footers.last();
+    footers.remove();
+    flexDivs.last().after(lastFooter);
+  }
+  if (flexDivs.last().parents().length > 0) {
```

Comment:
~~Add a comment to explain what this portion of the code is doing (something along the line of removing the divs that were wrapping the footer).~~ Or it can be a separate method on its own. EDIT: Think it should be a separate method, since this method is only concerned with removing duplicate footers.

Reply:
As you mentioned, also handle the case where the user does this:

```
<div>
  <div>
    <div>
      <div>
        <footer>
          The user wrapped so many div inside the footer, but shouldn't.
        </footer>
      </div>
    </div>
  </div>
</div>
```

Reply:
Handling a case like that would require a `while loop` that iteratively destroys parent html tags until it has no more parents.